jinja2/nativetypes.py

```diff
@@ -20,10 +21,13 @@ def native_concat(nodes):
     if not head:
         return None
 
+    if isinstance(nodes, types.GeneratorType):
```

Comment:
Let's keep this in the `else` block, no need to run the check if `len(head) == 1`.

Reply:
:+1: